### PR TITLE
Update "mapathon planning checklist" and training & event materials"

### PR DESCRIPTION
**Pull request by imbaah**
with access to logos

### CODE REVIEW

File: app/_data/en.yml [hunk 1]
```diff
@@ -395,7 +396,8 @@ host:
     title: "Training and Event Materials"
     text: "Everything you need to run a successful mapathon! (Google Drive folder link)"
     url:  "https://drive.google.com/drive/folders/1gXkvuQNRhfyOVv6XjmncFFUIK4m_yCTb?usp=sharing"
- 
+    text: "You can also download the Missing Maps logo in common formats, as well as a customizable logo to make one for your community!"
```
Comment (danbjoseph) [inline app/_data/en.yml:399]: you can only have 1 `host.materials.text` and 1  `host.materials.url`, this is the line of HTML where it is added in...
https://github.com/MissingMaps/missingmaps.github.io/blob/63696804c5189dc0f90ed1de606c0942f91ea23b/app/_includes/host.html#L81

some item lists are looped through, for example `host.materials.event_check_list` (as shown on the below line of HTML), but the above `text` and `url` isn't like that.
https://github.com/MissingMaps/missingmaps.github.io/blob/63696804c5189dc0f90ed1de606c0942f91ea23b/app/_includes/host.html#L66

**Review by danbjoseph — CHANGES_REQUESTED** [review]

File: app/_data/en.yml (hunk 1 above)
Comment (imbaah) [reply app/_data/en.yml:399]: First point I understand. The second one about the list however, I don't understand why they cannot be added. Would you be available sometime this week so I can show you want we want to update and you can tell me how I can/cannot? If you got time this week to have a short call, I'll send you slack msg.

Comment (danbjoseph) [reply app/_data/en.yml:399]: the looped list was just an example of a different way of adding content that is used in a different place on the website. for your edits, it is a 1-to-1 relationship between an item in the `.yml` language file and a placeholder on the website template. there is just one `host.materials.text` placeholder and so you can only have one `host.materials.text` value in the language file. your current change creates a second `host.materials.text` value in the language file. when the website it built, it won't know which of the 2 values from the language file to put into the 1 placeholder on the page template.

Comment (imbaah) [reply app/_data/en.yml:399]: Thank you for explaining the issue. I now understand the problem better but my lack of general GitHub knowledge doesn't allow me to come up with a solution. I sent you a slack msg, could we please meet whenever you're available or direct me to someone who could help me please? Thank you!

**Pull request merged**